src/Tizen.Network.Tethering/Tizen.Network.Tethering/ConnectionStateChangedEventArgs.cs

```diff
@@ -0,0 +1,42 @@
+using System;
+
+namespace Tizen.Network.Tethering
+{
+    /// <summary>
+    /// This class contains information related to Connection State.
```

Comment:
If an event args is designed for a single event then I like to have this event referenced in summary, but for this to work the event would need to be visible, now it is hidden. Then maybe this class should also be hidden.
```suggestion
    /// This class contains information related to <see cref="TetheringExtManager.ConnectionStateChanged"/> event.
```